pkg/utils/proc/status_test.go

```diff
@@ -72,12 +131,12 @@ func TestProcStatusParsing(t *testing.T) {
 			name: "Correct parsing of mock status file",
```

Comment:
I think maybe we should add some more tests here (if its the case).

Reply:
ditto.